client/src/components/Layout/UserHome/UserHome.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import Profile from '../../../Images/Ellipse 2.jpg'
+import Delete from '../../../Images/delete.jpg'
+import Edit from '../../../Images/Edit.png'
+import './UserHome.css'
+import axios from 'axios'
+class UserHome extends Component {
+  state = {
+    events :[]
+  }
+  componentDidMount () {
+       axios.get('/api/user-events')
```

Comment:
You should get the user events, so send parameters related to the user with the request